src/ImageSharp/ColorSpaces/Companding/SRgbCompanding.cs

```diff
@@ -18,49 +22,114 @@ namespace SixLabors.ImageSharp.ColorSpaces.Companding
     /// </remarks>
     public static class SRgbCompanding
     {
+        private const int Length = Scale + 2; // 256kb @ 16bit precision.
+        private const int Scale = (1 << 16) - 1;
+
+        private static readonly Lazy<float[]> LazyCompressTable = new Lazy<float[]>(() =>
+        {
+            var result = new float[Length];
+
+            for (int i = 0; i < result.Length; i++)
+            {
+                double d = (double)i / Scale;
+                if (d <= (0.04045 / 12.92))
+                {
+                    d *= 12.92;
+                }
+                else
+                {
+                    d = (1.055 * Math.Pow(d, 1.0 / 2.4)) - 0.055;
+                }
+
+                result[i] = (float)d;
+            }
+
+            return result;
+        });
+
+        private static readonly Lazy<float[]> LazyExpandTable = new Lazy<float[]>(() =>
+        {
+            var result = new float[Length];
+
+            for (int i = 0; i < result.Length; i++)
+            {
+                double d = (double)i / Scale;
+                if (d <= 0.04045)
+                {
+                    d /= 12.92;
+                }
+                else
+                {
+                    d = Math.Pow((d + 0.055) / 1.055, 2.4);
+                }
+
+                result[i] = (float)d;
+            }
+
+            return result;
+        });
+
+        private static readonly float[] ExpandTable = LazyExpandTable.Value;
```

Comment:
Why Lazy if this is a field?

Reply:
Yeah, doesn't need to be,. Will update.

Reply:
I would make `ExpandTable` a property.  The buffers seem to be huge, lazy init makes sense here. (16kbytes if I get it right)

Reply:
Updated.